plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1714,6 +1714,35 @@ class TestCollisionFrequencies:
         Coulomb_log=1 * u.dimensionless_unscaled,
     )
 
+    MKS_unit_conversion_test_constructor_arguments = {
```

Comment:
@pheuer If you could think of a better name for this that would be appreciated, I think this is probably fine though

Reply:
I think this is fine! A bit long, but it fits in the lines fine and is  very descriptive.